packages/generator-langium/templates/test/test/linking/linking.test.ts

```diff
@@ -0,0 +1,53 @@
+import { afterEach, beforeAll, describe, expect, test } from "<%= JestOrVitest %>";
+import { EmptyFileSystem, type LangiumDocument } from "langium";
+import { expandToString as s} from "langium";
+import { parseHelper } from "langium/test";
+import { create<%= LanguageName %>Services } from "../../src/language/<%= language-id %>-module.js";
+import { Model, isModel } from "../../src/language/generated/ast.js";
+
+let services: ReturnType<typeof create<%= LanguageName %>Services>;
+let parse:    ReturnType<typeof parseHelper<Model>>;
+let document: LangiumDocument<Model> | undefined;
+
+beforeAll(async () => {
+    services = create<%= LanguageName %>Services(EmptyFileSystem);
+    parse = parseHelper<Model>(services.<%= LanguageName %>);
+
+    // activate the following if your linking test requires elements from a built-in library, for example
+    // await services.shared.workspace.WorkspaceManager.initializeWorkspace([]);
+});
+
+afterEach(async () => {
+    document && await services.shared.workspace.DocumentBuilder.update([], [ document?.uri ]);
```

Comment:
Suggestion: We have a method in our testing functions for that.

```suggestion
    clearDocuments(services);
```

Reply:
Added.